.github/workflows/upgrade_downgrade_test_backups_e2e.yml

```diff
@@ -44,13 +44,13 @@ jobs:
         run: |
           previous_release_ref=$(./tools/get_previous_release.sh ${{github.base_ref}} ${{github.ref}})
           echo $previous_release_ref
-          echo "::set-output name=previous_release_ref::${previous_release_ref}"
+          echo "previous_release_ref=${previous_release_ref}" >> $GITHUB_OUTPUT
 
   upgrade_downgrade_test_e2e:
     timeout-minutes: 60
     if: always() && (needs.get_previous_release.result == 'success')
     name: Run Upgrade Downgrade Test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
This might be delibrate.. downgraded version doesn't handled backing-up mysql8.0 correctly .. We will see if the test fails..

Reply:
@rsajwani The goal of upgrading was that we can still run CI when GitHub removes 18.04 support. So we only have 2 choices, either we upgrade or we remove the CI job. We can't leave it on 18.04 since that will stop working then. 

We can also look at forcing a MySQL version earlier than 8.0.30 to fix this?

Reply:
@dbussink , did you check all the CI workflows were passing before pushing this PR ?? 
ideally we shouldn't have pushed this PR until all the test are passing. I can see Run Upgrade Downgrade Test was failing...